src/os_posix.cc

```diff
@@ -255,8 +255,20 @@ void OS::out_of_memory(const char* reason) {
   abort();
 }
 
-const char* OS::getenv(const char* variable) {
-  return ::getenv(variable);
+char* OS::getenv(const char* variable) {
+  // TODO(florian): getenv is not reentrant.
```

Comment:
That is really easy to do. We have lots of global mutex dying to be used from a Locker.